src/content/en/2021/cdn.md

```diff
@@ -252,49 +250,51 @@ Similar trend is observed for the third party domains below. These web services
 
-This is quite important when you look at the fact that all sites will have to be on TLS these days.  Optimal performance at this layer is essential for other steps that follow TLS connection. In this regard, CDNs are able to move more users to lower percentile brackets compared to Origin.
+This is quite important when you consider that all sites will have to be on TLS these days. Optimal performance at this layer is essential for other steps that follow TLS connection. In this regard, CDNs are able to move more users to lower percentile brackets compared to direct origin connections.
 
 ## HTTP/2+ (HTTP/2 or better) adoption
 
-HTTP/2 was introduced with a lot of hype and expectation. This was because the application layer protocol has not been updated since HTTP 1.1 in 1997. Since then, the web traffic trend, content-type, content size, website design, platforms, mobile apps etc., have evolved significantly. Thus, there was a need to have a protocol which can meet the requirements of the modern day web traffic and that protocol was going to be HTTP/2.
+HTTP/2 was introduced with a lot of hype and expectation. This was because the application layer protocol had not been updated since HTTP 1.1 in 1997. Since then, the web traffic trend, content-type, content size, website design, platforms, mobile apps and more have evolved significantly. Thus, there was a need to have a protocol which can meet the requirements of the modern-day web traffic and that protocol was realized with HTTP/2, and then further improved with the more recent HTTP/3.
 
-However, the implementation challenges of HTTP/2 features discouraged adoption. In addition, the net performance gains which can be expected with these changes was also not clear.
+However, the implementation challenges of HTTP/2 discouraged adoption. In addition, the net performance gains which can be expected with these changes was also not clear. Challenges repeated with the introduction of HTTP/3.
 
-This was where the CDNs became an intermediary in bridging the challenge of HTTP/2 implementation for web owners. Similar to TLS, an HTTP/2 connection terminates at the CDN level and this provides web owners the ability to deliver their website and subdomains over HTTP/2 without the need to upgrade their infrastructure to support it.
+This was where the CDNs being the intermediary can help in bridging the challenge of HTTP/2 implementation for web owners. An HTTP/2 connection terminates at the CDN level, and this provides web owners the ability to deliver their website and subdomains over HTTP/2 without the need to upgrade their infrastructure to support it—the exact same reasons and benefits we saw for newer TLS versions.
```

Comment:
Tweaked this slightly to introduce HTTP/3. Let me know if you want to change.

Reply:
Works for me Barry. I'm good with these tweaks